src/braket/aws/aws_device.py

```diff
@@ -182,7 +182,7 @@ def run(
 
         See Also:
             `braket.aws.aws_quantum_task.AwsQuantumTask.create()`
-        """
+        """  # noqa E501
```

Comment:
I don't understand this noqa (line too long). Does it apply to the whole doc string? Why can't we fix it here?

Reply:
It does apply to the whole docstring. I am not aware of a way to have the parser read the types on this object with a new line added. One alternative is to define a type which consists of the union on L130 and then apply that type there.

Reply:
I imagine this is mostly geared towards the docs site. I am not sure how external types get rendered there but that can be a follow up.